src/common/src/config.rs

```diff
@@ -200,6 +200,10 @@ pub struct StorageConfig {
     /// Max sub compaction task numbers
     #[serde(default = "default::max_sub_compaction")]
     pub max_sub_compaction: u32,
+
+    /// Whether to enable state_store_v2 for hummock
+    #[serde(default = "default::enable_state_store_v2")]
+    pub enable_state_store_v2: bool,
```

Comment:
In the future by default we will use the v2, and therefore it's better to rename it to `enable_state_store_v1` and set the default setting to false, so that when we want to enable v1, we can set an option whose name is related to v1.